src/Encryption/index.js

```diff
@@ -171,7 +171,13 @@ class Encryption {
    * @public
    */
   base64Encode (unencoded) {
-    return Buffer.from(unencoded || '').toString('base64')
+    if (typeof Buffer.from === 'function') {
```

Comment:
@thetutlage I'm not sure what the best way to handle older versions of node here is.